src/server/middleware/missingBundleMiddleware.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ */
+const MultiCompiler = require('webpack/lib/MultiCompiler');
+const dedent = require('dedent');
+
+function missingBundleMiddleware(compiler) {
+  return (req, res, next) => {
+    // In multi compiler mode, all entry points should be available
+    if (compiler instanceof MultiCompiler) {
```

Comment:
What if it's a different error and not availability of the entry point? (the second point below)